bdk-ffi/src/tx_builder.rs

```diff
@@ -289,13 +289,13 @@ impl TxBuilder {
     /// This will be used to:
     ///
     /// 1. Set the `nLockTime` for preventing fee sniping. Note: This will be ignored if you manually specify a
-    /// `nlocktime` using `TxBuilder::nlocktime`.
+    ///    `nlocktime` using `TxBuilder::nlocktime`.
```

Comment:
Is the extra spacing here and in a couple other spots in the diff for any specific reason?

Reply:
Yeah those were warnings from cargo's linter (the `fmt` tool). I can't find the commit right now but do you remember when we had to turn off the "fail" on the fmt CI workflow when warnings were issued, and only fail on fmt's failures?

This had the downside of now letting us merge stuff even if there is a warning by cargo's linter. Not a big deal, but in this case I just thought I'd deal with the warning. The commit that did that is 78f38cff62fe45d8be3e5eae5281e668ca13a0ec.